contracts/ocr2/src/contract.rs

```diff
@@ -929,11 +931,15 @@ pub fn execute_set_billing(
         Event::new("set_billing")
             .add_attribute(
                 "recommended_gas_price",
```

Comment:
Should we rename this attribute as `recommended_gas_price_uluna` to keep it consistent?